src/i_main.c

```diff
@@ -41,6 +41,17 @@ int main(int argc, char **argv)
     myargc = argc;
     myargv = argv;
 
+    //!
+    // @arg
+    //
+    // Print the program version and exit.
+    //
+
+    if (M_ParmExists("-version") || M_ParmExists("--version")) {
+        I_Error (PACKAGE_NAME " " PACKAGE_VERSION);
+        exit(0);
```

Comment:
`I_Error` never returns, the `exit` call is redundant.

Someone should mark `I_Error` with GCC's `noreturn` attribute and then CI should notice things like this for us.

Reply:
Thanks